File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var sumOfUnique = function(nums) {
  let obj = {};
  let sum = 0;
  for (const num of nums) {
    obj[num] = (obj[num] || 0) + 1;
  }

  for (const key in obj) {
    if (obj[key] === 1) sum += +key;
  }
  return sum;
};

File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.py
class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        freq_dict = {}
        count = 0
        for num in nums:
            freq_dict[num] = freq_dict.get(num, 0) + 1
            
        for num in nums:
            if freq_dict[num] == 1:
                count += num
        return count